src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_PatternSwitchStatement.cs

```diff
@@ -320,7 +320,7 @@ private void AddBindings(ArrayBuilder<BoundStatement> sectionBuilder, ImmutableA
                 {
                     var source = kv.Key;
                     var dest = kv.Value;
```

Comment:
Thank you for making this change.

---
In reply to: [80849000](https://github.com/dotnet/roslyn/pull/13999#discussion_r80849000) [](ancestors = 80849000,80815417,80800519)